packages/plugin-core/src/workspace/tutorialInitializer.ts

```diff
@@ -61,36 +54,28 @@ export class TutorialInitializer
 
     const vpath = vault2Path({ vault: opts.vaults[0], wsRoot: opts.wsRoot });
 
-    fs.copySync(path.join(dendronWSTemplate.fsPath, "tutorial"), vpath);
-
-    const ABUserGroup = MEETING_NOTE_TUTORIAL_TEST.getUserGroup(
+    const ABUserGroup = AB_2022_05_REFACTOR_MENTION.getUserGroup(
       SegmentClient.instance().anonymousId
     );
 
-    const replaceString =
-      ABUserGroup === MeetingNoteTestGroups.show ? MeetingNoteTutorialText : "";
-
-    // Tailor the tutorial text to the particular OS and for their workspace location.
-    const options = {
-      files: [path.join(vpath, "*.md")],
-
-      from: [/%KEYBINDING%/g, /%WORKSPACE_ROOT%/g, /%MEETING_NOTE_CONTENT%/g],
-      to: [
-        process.platform === "darwin" ? "Cmd" : "Ctrl",
-        path.join(opts.wsRoot, "dendron.code-workspace"),
-        replaceString,
-      ],
-    };
-
-    rif.replaceInFile(options).catch((err: Error) => {
-      Logger.error({
-        ctx,
-        error: new DendronError({
-          innerError: err,
-          message: "error replacing tutorial placeholder text",
-        }),
-      });
-    });
+    switch (ABUserGroup) {
```

Comment:
also - for the initial tutorial sync, can we also include the original tutorial with the meeting notes addition as an alternative? this way, we can compare the current tutorial as a baseline, compared to updated tutorial and updated tutorial without refactoring

Reply:
As in, have three groups? e.g.) `"mention" | "noMention" | "original"`

Reply:
In this case I think we should also include the meeting note paragraph in the other two tutorials.

Reply:
Agree with all comments. 

So then we can understand
- Difference between 'old' (current) and 'new' (wiki.dendron.so version)
- Difference between 'refactor mention' and 'no-refactor-mention'

One thing to keep in mind is that if we have 3 groups, we should expect 1.5x the amount of time to see the same amount of data, and 1.5x the amount of time to see results with the same statistical significance as 2 groups.

Reply:
> One thing to keep in mind is that if we have 3 groups, we should expect 1.5x the amount of time to see the same amount of data, and 1.5x the amount of time to see results with the same statistical significance as 2 groups.

Noted. I know @kevinslin is planning to set up a short/long tutorial AB test to come up next. Are you fine with this test running for the next 2 releases?

Reply:
yep